src/loaders/ObjectLoader.js

```diff
@@ -261,6 +263,12 @@ class ObjectLoader extends Loader {
 
 	}
 
+	registerGeometry( type, klass ) {
```

Comment:
@mrdoob Are you okay with this kind of new interface?

Reply:
I updated a merge from mrdoob/dev so there should be no conflicts now